includes/data-sources-sdks-server-side.md

```diff
@@ -0,0 +1,11 @@
+<!-- To add an entry, first add an SVG logo in overrides/.icons, then add a new line item in the table. Wrap the icon filename in colons to reference it. -->
+
+<div class="grid cards" markdown>
+
+- :java: [Java](../../data/sdks/java/)
+- :node: [Node.js](../../data/sdks/typescript-node/)
+- :python: [Python](../../data/sdks/python/)
+- :react: [React Native](../../data/sdks/typescript-react-native/)
```

Comment:
React Native is a client-side SDK, should be removed from this section of the list

Reply:
AH I knew there was at least one dumb copy-paste error left somewhere. Thanks!